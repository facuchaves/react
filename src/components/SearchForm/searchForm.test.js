// import renderer from 'react-test-renderer';
import Enzyme, {shallow} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
// import {cleanup, fireEvent, render} from '@testing-library/react';
import {SearchFormHome} from "./index";
import {expect} from "chai";
const sinon = require("sinon");

Enzyme.configure({adapter: new Adapter()});

test('searchForm', () => {

    const handleSubmit = () => {}

    const callback = sinon.spy();

    // const handleSubmitSpies = jest.spyOn(handleSubmit);
    
    const searchFormHome = shallow(<SearchFormHome handleSubmit={callback} />);

    const form = searchFormHome.find('[test-id="searchIssueFromTestId"]')
    expect(form.props().onSubmit).to.have.lengthOf(1);

    form.simulate('submit',{ preventDefault : () => {} })
    expect(callback.called).to.be.true;

//   // manually trigger the callback
//   renderer.act(() => {
//     tree.props.onMouseEnter();
//   });
//   // re-rendering
//   tree = component.toJSON();
//   expect(tree).toMatchSnapshot();

//   // manually trigger the callback
//   renderer.act(() => {
//     tree.props.onMouseLeave();
//   });
//   // re-rendering
//   tree = component.toJSON();
//   expect(tree).toMatchSnapshot();
});