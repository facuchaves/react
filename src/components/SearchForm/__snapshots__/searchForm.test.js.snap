// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`searchForm 1`] = `
<div>
  <form
    onSubmit={[Function]}
    style={
      Object {
        "display": "inline-flex",
      }
    }
  >
    <style
      dangerouslySetInnerHTML={
        Object {
          "__html": ".css-1u3bzj6-MuiFormControl-root-MuiTextField-root{display:-webkit-inline-box;display:-webkit-inline-flex;display:-ms-inline-flexbox;display:inline-flex;-webkit-flex-direction:column;-ms-flex-direction:column;flex-direction:column;position:relative;min-width:0;padding:0;margin:0;border:0;vertical-align:top;}",
        }
      }
      data-emotion="css 1u3bzj6-MuiFormControl-root-MuiTextField-root"
    />
    <div
      className="MuiFormControl-root MuiTextField-root css-1u3bzj6-MuiFormControl-root-MuiTextField-root"
    >
      <style
        dangerouslySetInnerHTML={
          Object {
            "__html": ".css-1x92c1k-MuiInputLabel-root{display:block;transform-origin:top left;white-space:nowrap;overflow:hidden;text-overflow:ellipsis;max-width:calc(100% - 24px);position:absolute;left:0;top:0;-webkit-transform:translate(14px, 16px) scale(1);-moz-transform:translate(14px, 16px) scale(1);-ms-transform:translate(14px, 16px) scale(1);transform:translate(14px, 16px) scale(1);-webkit-transition:color 200ms cubic-bezier(0.0, 0, 0.2, 1) 0ms,-webkit-transform 200ms cubic-bezier(0.0, 0, 0.2, 1) 0ms,max-width 200ms cubic-bezier(0.0, 0, 0.2, 1) 0ms;transition:color 200ms cubic-bezier(0.0, 0, 0.2, 1) 0ms,transform 200ms cubic-bezier(0.0, 0, 0.2, 1) 0ms,max-width 200ms cubic-bezier(0.0, 0, 0.2, 1) 0ms;z-index:1;pointer-events:none;}",
          }
        }
        data-emotion="css 1x92c1k-MuiInputLabel-root"
      />
      <style
        dangerouslySetInnerHTML={
          Object {
            "__html": ".css-14s5rfu-MuiFormLabel-root-MuiInputLabel-root{color:rgba(0, 0, 0, 0.6);font-family:\\"Roboto\\",\\"Helvetica\\",\\"Arial\\",sans-serif;font-weight:400;font-size:1rem;line-height:1.4375em;letter-spacing:0.00938em;padding:0;position:relative;display:block;transform-origin:top left;white-space:nowrap;overflow:hidden;text-overflow:ellipsis;max-width:calc(100% - 24px);position:absolute;left:0;top:0;-webkit-transform:translate(14px, 16px) scale(1);-moz-transform:translate(14px, 16px) scale(1);-ms-transform:translate(14px, 16px) scale(1);transform:translate(14px, 16px) scale(1);-webkit-transition:color 200ms cubic-bezier(0.0, 0, 0.2, 1) 0ms,-webkit-transform 200ms cubic-bezier(0.0, 0, 0.2, 1) 0ms,max-width 200ms cubic-bezier(0.0, 0, 0.2, 1) 0ms;transition:color 200ms cubic-bezier(0.0, 0, 0.2, 1) 0ms,transform 200ms cubic-bezier(0.0, 0, 0.2, 1) 0ms,max-width 200ms cubic-bezier(0.0, 0, 0.2, 1) 0ms;z-index:1;pointer-events:none;}.css-14s5rfu-MuiFormLabel-root-MuiInputLabel-root.Mui-focused{color:#1976d2;}.css-14s5rfu-MuiFormLabel-root-MuiInputLabel-root.Mui-disabled{color:rgba(0, 0, 0, 0.38);}.css-14s5rfu-MuiFormLabel-root-MuiInputLabel-root.Mui-error{color:#d32f2f;}",
          }
        }
        data-emotion="css 14s5rfu-MuiFormLabel-root-MuiInputLabel-root"
      />
      <label
        className="MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined MuiFormLabel-root MuiFormLabel-colorPrimary css-14s5rfu-MuiFormLabel-root-MuiInputLabel-root"
        data-shrink={false}
        htmlFor="outlined-basic"
        id="outlined-basic-label"
      >
        Outlined
      </label>
      <style
        dangerouslySetInnerHTML={
          Object {
            "__html": "@-webkit-keyframes mui-auto-fill{from{display:block;}}@keyframes mui-auto-fill{from{display:block;}}@-webkit-keyframes mui-auto-fill-cancel{from{display:block;}}@keyframes mui-auto-fill-cancel{from{display:block;}}",
          }
        }
        data-emotion="css-global 1prfaxn"
      />
      <style
        dangerouslySetInnerHTML={
          Object {
            "__html": ".css-9ddj71-MuiInputBase-root-MuiOutlinedInput-root{font-family:\\"Roboto\\",\\"Helvetica\\",\\"Arial\\",sans-serif;font-weight:400;font-size:1rem;line-height:1.4375em;letter-spacing:0.00938em;color:rgba(0, 0, 0, 0.87);box-sizing:border-box;position:relative;cursor:text;display:-webkit-inline-box;display:-webkit-inline-flex;display:-ms-inline-flexbox;display:inline-flex;-webkit-align-items:center;-webkit-box-align:center;-ms-flex-align:center;align-items:center;position:relative;border-radius:4px;}.css-9ddj71-MuiInputBase-root-MuiOutlinedInput-root.Mui-disabled{color:rgba(0, 0, 0, 0.38);cursor:default;}.css-9ddj71-MuiInputBase-root-MuiOutlinedInput-root:hover .MuiOutlinedInput-notchedOutline{border-color:rgba(0, 0, 0, 0.87);}@media (hover: none){.css-9ddj71-MuiInputBase-root-MuiOutlinedInput-root:hover .MuiOutlinedInput-notchedOutline{border-color:rgba(0, 0, 0, 0.23);}}.css-9ddj71-MuiInputBase-root-MuiOutlinedInput-root.Mui-focused .MuiOutlinedInput-notchedOutline{border-color:#1976d2;border-width:2px;}.css-9ddj71-MuiInputBase-root-MuiOutlinedInput-root.Mui-error .MuiOutlinedInput-notchedOutline{border-color:#d32f2f;}.css-9ddj71-MuiInputBase-root-MuiOutlinedInput-root.Mui-disabled .MuiOutlinedInput-notchedOutline{border-color:rgba(0, 0, 0, 0.26);}",
          }
        }
        data-emotion="css 9ddj71-MuiInputBase-root-MuiOutlinedInput-root"
      />
      <div
        className="MuiOutlinedInput-root MuiInputBase-root MuiInputBase-colorPrimary MuiInputBase-formControl css-9ddj71-MuiInputBase-root-MuiOutlinedInput-root"
        onClick={[Function]}
      >
        <style
          dangerouslySetInnerHTML={
            Object {
              "__html": ".css-1t8l2tu-MuiInputBase-input-MuiOutlinedInput-input{font:inherit;letter-spacing:inherit;color:currentColor;padding:4px 0 5px;border:0;box-sizing:content-box;background:none;height:1.4375em;margin:0;-webkit-tap-highlight-color:transparent;display:block;min-width:0;width:100%;-webkit-animation-name:mui-auto-fill-cancel;animation-name:mui-auto-fill-cancel;-webkit-animation-duration:10ms;animation-duration:10ms;padding:16.5px 14px;}.css-1t8l2tu-MuiInputBase-input-MuiOutlinedInput-input::-webkit-input-placeholder{color:currentColor;opacity:0.42;-webkit-transition:opacity 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;transition:opacity 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;}.css-1t8l2tu-MuiInputBase-input-MuiOutlinedInput-input::-moz-placeholder{color:currentColor;opacity:0.42;-webkit-transition:opacity 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;transition:opacity 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;}.css-1t8l2tu-MuiInputBase-input-MuiOutlinedInput-input:-ms-input-placeholder{color:currentColor;opacity:0.42;-webkit-transition:opacity 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;transition:opacity 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;}.css-1t8l2tu-MuiInputBase-input-MuiOutlinedInput-input::-ms-input-placeholder{color:currentColor;opacity:0.42;-webkit-transition:opacity 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;transition:opacity 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;}.css-1t8l2tu-MuiInputBase-input-MuiOutlinedInput-input:focus{outline:0;}.css-1t8l2tu-MuiInputBase-input-MuiOutlinedInput-input:invalid{box-shadow:none;}.css-1t8l2tu-MuiInputBase-input-MuiOutlinedInput-input::-webkit-search-decoration{-webkit-appearance:none;}label[data-shrink=false]+.MuiInputBase-formControl .css-1t8l2tu-MuiInputBase-input-MuiOutlinedInput-input::-webkit-input-placeholder{opacity:0!important;}label[data-shrink=false]+.MuiInputBase-formControl .css-1t8l2tu-MuiInputBase-input-MuiOutlinedInput-input::-moz-placeholder{opacity:0!important;}label[data-shrink=false]+.MuiInputBase-formControl .css-1t8l2tu-MuiInputBase-input-MuiOutlinedInput-input:-ms-input-placeholder{opacity:0!important;}label[data-shrink=false]+.MuiInputBase-formControl .css-1t8l2tu-MuiInputBase-input-MuiOutlinedInput-input::-ms-input-placeholder{opacity:0!important;}label[data-shrink=false]+.MuiInputBase-formControl .css-1t8l2tu-MuiInputBase-input-MuiOutlinedInput-input:focus::-webkit-input-placeholder{opacity:0.42;}label[data-shrink=false]+.MuiInputBase-formControl .css-1t8l2tu-MuiInputBase-input-MuiOutlinedInput-input:focus::-moz-placeholder{opacity:0.42;}label[data-shrink=false]+.MuiInputBase-formControl .css-1t8l2tu-MuiInputBase-input-MuiOutlinedInput-input:focus:-ms-input-placeholder{opacity:0.42;}label[data-shrink=false]+.MuiInputBase-formControl .css-1t8l2tu-MuiInputBase-input-MuiOutlinedInput-input:focus::-ms-input-placeholder{opacity:0.42;}.css-1t8l2tu-MuiInputBase-input-MuiOutlinedInput-input.Mui-disabled{opacity:1;-webkit-text-fill-color:rgba(0, 0, 0, 0.38);}.css-1t8l2tu-MuiInputBase-input-MuiOutlinedInput-input:-webkit-autofill{-webkit-animation-duration:5000s;animation-duration:5000s;-webkit-animation-name:mui-auto-fill;animation-name:mui-auto-fill;}.css-1t8l2tu-MuiInputBase-input-MuiOutlinedInput-input:-webkit-autofill{border-radius:inherit;}",
            }
          }
          data-emotion="css 1t8l2tu-MuiInputBase-input-MuiOutlinedInput-input"
        />
        <input
          aria-invalid={false}
          autoFocus={false}
          className="MuiOutlinedInput-input MuiInputBase-input css-1t8l2tu-MuiInputBase-input-MuiOutlinedInput-input"
          disabled={false}
          id="outlined-basic"
          name="q"
          onAnimationStart={[Function]}
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          required={false}
          type="text"
          value=""
        />
        <style
          dangerouslySetInnerHTML={
            Object {
              "__html": ".css-rki20w-MuiOutlinedInput-notchedOutline{border-color:rgba(0, 0, 0, 0.23);}",
            }
          }
          data-emotion="css rki20w-MuiOutlinedInput-notchedOutline"
        />
        <style
          dangerouslySetInnerHTML={
            Object {
              "__html": ".css-1d3z3hw-MuiOutlinedInput-notchedOutline{text-align:left;position:absolute;bottom:0;right:0;top:-5px;left:0;margin:0;padding:0 8px;pointer-events:none;border-radius:inherit;border-style:solid;border-width:1px;overflow:hidden;min-width:0%;border-color:rgba(0, 0, 0, 0.23);}",
            }
          }
          data-emotion="css 1d3z3hw-MuiOutlinedInput-notchedOutline"
        />
        <fieldset
          aria-hidden={true}
          className="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
        >
          <style
            dangerouslySetInnerHTML={
              Object {
                "__html": ".css-1ftyaf0{float:unset;overflow:hidden;display:block;width:auto;padding:0;height:11px;font-size:0.75em;visibility:hidden;max-width:0.01px;-webkit-transition:max-width 50ms cubic-bezier(0.0, 0, 0.2, 1) 0ms;transition:max-width 50ms cubic-bezier(0.0, 0, 0.2, 1) 0ms;white-space:nowrap;}.css-1ftyaf0>span{padding-left:5px;padding-right:5px;display:inline-block;opacity:0;visibility:visible;}",
              }
            }
            data-emotion="css 1ftyaf0"
          />
          <legend
            className="css-1ftyaf0"
          >
            <span>
              Outlined
            </span>
          </legend>
        </fieldset>
      </div>
    </div>
    <style
      dangerouslySetInnerHTML={
        Object {
          "__html": ".css-ahj2mt-MuiTypography-root{margin:0;font-family:\\"Roboto\\",\\"Helvetica\\",\\"Arial\\",sans-serif;font-weight:400;font-size:1rem;line-height:1.5;letter-spacing:0.00938em;}",
        }
      }
      data-emotion="css ahj2mt-MuiTypography-root"
    />
    <p
      className="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
    >
      Closed
    </p>
    <style
      dangerouslySetInnerHTML={
        Object {
          "__html": ".css-julti5-MuiSwitch-root{display:-webkit-inline-box;display:-webkit-inline-flex;display:-ms-inline-flexbox;display:inline-flex;width:58px;height:38px;overflow:hidden;padding:12px;box-sizing:border-box;position:relative;-webkit-flex-shrink:0;-ms-flex-negative:0;flex-shrink:0;z-index:0;vertical-align:middle;}@media print{.css-julti5-MuiSwitch-root{-webkit-print-color-adjust:exact;color-adjust:exact;}}",
        }
      }
      data-emotion="css julti5-MuiSwitch-root"
    />
    <span
      className="MuiSwitch-root MuiSwitch-sizeMedium css-julti5-MuiSwitch-root"
    >
      <style
        dangerouslySetInnerHTML={
          Object {
            "__html": ".css-ktn406-MuiSwitch-switchBase{position:absolute;top:0;left:0;z-index:1;color:#fff;-webkit-transition:left 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,-webkit-transform 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;transition:left 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,transform 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;}.css-ktn406-MuiSwitch-switchBase.Mui-checked{-webkit-transform:translateX(20px);-moz-transform:translateX(20px);-ms-transform:translateX(20px);transform:translateX(20px);}.css-ktn406-MuiSwitch-switchBase.Mui-disabled{color:#f5f5f5;}.css-ktn406-MuiSwitch-switchBase.Mui-checked+.MuiSwitch-track{opacity:0.5;}.css-ktn406-MuiSwitch-switchBase.Mui-disabled+.MuiSwitch-track{opacity:0.12;}.css-ktn406-MuiSwitch-switchBase .MuiSwitch-input{left:-100%;width:300%;}.css-ktn406-MuiSwitch-switchBase:hover{background-color:rgba(0, 0, 0, 0.04);}@media (hover: none){.css-ktn406-MuiSwitch-switchBase:hover{background-color:transparent;}}.css-ktn406-MuiSwitch-switchBase.Mui-checked{color:#1976d2;}.css-ktn406-MuiSwitch-switchBase.Mui-checked:hover{background-color:rgba(25, 118, 210, 0.04);}@media (hover: none){.css-ktn406-MuiSwitch-switchBase.Mui-checked:hover{background-color:transparent;}}.css-ktn406-MuiSwitch-switchBase.Mui-checked.Mui-disabled{color:rgb(167, 202, 237);}.css-ktn406-MuiSwitch-switchBase.Mui-checked+.MuiSwitch-track{background-color:#1976d2;}",
          }
        }
        data-emotion="css ktn406-MuiSwitch-switchBase"
      />
      <style
        dangerouslySetInnerHTML={
          Object {
            "__html": ".css-dsp0n2-MuiSwitch-switchBase{padding:9px;border-radius:50%;position:absolute;top:0;left:0;z-index:1;color:#fff;-webkit-transition:left 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,-webkit-transform 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;transition:left 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,transform 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;}.css-dsp0n2-MuiSwitch-switchBase.Mui-checked{-webkit-transform:translateX(20px);-moz-transform:translateX(20px);-ms-transform:translateX(20px);transform:translateX(20px);}.css-dsp0n2-MuiSwitch-switchBase.Mui-disabled{color:#f5f5f5;}.css-dsp0n2-MuiSwitch-switchBase.Mui-checked+.MuiSwitch-track{opacity:0.5;}.css-dsp0n2-MuiSwitch-switchBase.Mui-disabled+.MuiSwitch-track{opacity:0.12;}.css-dsp0n2-MuiSwitch-switchBase .MuiSwitch-input{left:-100%;width:300%;}.css-dsp0n2-MuiSwitch-switchBase:hover{background-color:rgba(0, 0, 0, 0.04);}@media (hover: none){.css-dsp0n2-MuiSwitch-switchBase:hover{background-color:transparent;}}.css-dsp0n2-MuiSwitch-switchBase.Mui-checked{color:#1976d2;}.css-dsp0n2-MuiSwitch-switchBase.Mui-checked:hover{background-color:rgba(25, 118, 210, 0.04);}@media (hover: none){.css-dsp0n2-MuiSwitch-switchBase.Mui-checked:hover{background-color:transparent;}}.css-dsp0n2-MuiSwitch-switchBase.Mui-checked.Mui-disabled{color:rgb(167, 202, 237);}.css-dsp0n2-MuiSwitch-switchBase.Mui-checked+.MuiSwitch-track{background-color:#1976d2;}",
          }
        }
        data-emotion="css dsp0n2-MuiSwitch-switchBase"
      />
      <style
        dangerouslySetInnerHTML={
          Object {
            "__html": ".css-5ryogn-MuiButtonBase-root-MuiSwitch-switchBase{display:-webkit-inline-box;display:-webkit-inline-flex;display:-ms-inline-flexbox;display:inline-flex;-webkit-align-items:center;-webkit-box-align:center;-ms-flex-align:center;align-items:center;-webkit-box-pack:center;-ms-flex-pack:center;-webkit-justify-content:center;justify-content:center;position:relative;box-sizing:border-box;-webkit-tap-highlight-color:transparent;background-color:transparent;outline:0;border:0;margin:0;border-radius:0;padding:0;cursor:pointer;-webkit-user-select:none;-moz-user-select:none;-ms-user-select:none;user-select:none;vertical-align:middle;-moz-appearance:none;-webkit-appearance:none;-webkit-text-decoration:none;text-decoration:none;color:inherit;padding:9px;border-radius:50%;position:absolute;top:0;left:0;z-index:1;color:#fff;-webkit-transition:left 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,-webkit-transform 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;transition:left 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,transform 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;}.css-5ryogn-MuiButtonBase-root-MuiSwitch-switchBase::-moz-focus-inner{border-style:none;}.css-5ryogn-MuiButtonBase-root-MuiSwitch-switchBase.Mui-disabled{pointer-events:none;cursor:default;}@media print{.css-5ryogn-MuiButtonBase-root-MuiSwitch-switchBase{-webkit-print-color-adjust:exact;color-adjust:exact;}}.css-5ryogn-MuiButtonBase-root-MuiSwitch-switchBase.Mui-checked{-webkit-transform:translateX(20px);-moz-transform:translateX(20px);-ms-transform:translateX(20px);transform:translateX(20px);}.css-5ryogn-MuiButtonBase-root-MuiSwitch-switchBase.Mui-disabled{color:#f5f5f5;}.css-5ryogn-MuiButtonBase-root-MuiSwitch-switchBase.Mui-checked+.MuiSwitch-track{opacity:0.5;}.css-5ryogn-MuiButtonBase-root-MuiSwitch-switchBase.Mui-disabled+.MuiSwitch-track{opacity:0.12;}.css-5ryogn-MuiButtonBase-root-MuiSwitch-switchBase .MuiSwitch-input{left:-100%;width:300%;}.css-5ryogn-MuiButtonBase-root-MuiSwitch-switchBase:hover{background-color:rgba(0, 0, 0, 0.04);}@media (hover: none){.css-5ryogn-MuiButtonBase-root-MuiSwitch-switchBase:hover{background-color:transparent;}}.css-5ryogn-MuiButtonBase-root-MuiSwitch-switchBase.Mui-checked{color:#1976d2;}.css-5ryogn-MuiButtonBase-root-MuiSwitch-switchBase.Mui-checked:hover{background-color:rgba(25, 118, 210, 0.04);}@media (hover: none){.css-5ryogn-MuiButtonBase-root-MuiSwitch-switchBase.Mui-checked:hover{background-color:transparent;}}.css-5ryogn-MuiButtonBase-root-MuiSwitch-switchBase.Mui-checked.Mui-disabled{color:rgb(167, 202, 237);}.css-5ryogn-MuiButtonBase-root-MuiSwitch-switchBase.Mui-checked+.MuiSwitch-track{background-color:#1976d2;}",
          }
        }
        data-emotion="css 5ryogn-MuiButtonBase-root-MuiSwitch-switchBase"
      />
      <span
        className="MuiSwitch-switchBase MuiSwitch-colorPrimary MuiButtonBase-root MuiSwitch-switchBase MuiSwitch-colorPrimary PrivateSwitchBase-root Mui-checked css-5ryogn-MuiButtonBase-root-MuiSwitch-switchBase"
        onBlur={[Function]}
        onClick={[Function]}
        onContextMenu={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={null}
      >
        <style
          dangerouslySetInnerHTML={
            Object {
              "__html": ".css-1m9pwf3{cursor:inherit;position:absolute;opacity:0;width:100%;height:100%;top:0;left:0;margin:0;padding:0;z-index:1;}",
            }
          }
          data-emotion="css 1m9pwf3"
        />
        <input
          aria-label="Switch demo"
          className="MuiSwitch-input PrivateSwitchBase-input css-1m9pwf3"
          defaultChecked={true}
          name="state"
          onChange={[Function]}
          type="checkbox"
          value="OPEN"
        />
        <style
          dangerouslySetInnerHTML={
            Object {
              "__html": ".css-jsexje-MuiSwitch-thumb{box-shadow:0px 2px 1px -1px rgba(0,0,0,0.2),0px 1px 1px 0px rgba(0,0,0,0.14),0px 1px 3px 0px rgba(0,0,0,0.12);background-color:currentColor;width:20px;height:20px;border-radius:50%;}",
            }
          }
          data-emotion="css jsexje-MuiSwitch-thumb"
        />
        <span
          className="MuiSwitch-thumb css-jsexje-MuiSwitch-thumb"
        />
      </span>
      <style
        dangerouslySetInnerHTML={
          Object {
            "__html": ".css-1yjjitx-MuiSwitch-track{height:100%;width:100%;border-radius:7px;z-index:-1;-webkit-transition:opacity 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,background-color 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;transition:opacity 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,background-color 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;background-color:#000;opacity:0.38;}",
          }
        }
        data-emotion="css 1yjjitx-MuiSwitch-track"
      />
      <span
        className="MuiSwitch-track css-1yjjitx-MuiSwitch-track"
      />
    </span>
    <style
      dangerouslySetInnerHTML={
        Object {
          "__html": ".css-ahj2mt-MuiTypography-root{margin:0;font-family:\\"Roboto\\",\\"Helvetica\\",\\"Arial\\",sans-serif;font-weight:400;font-size:1rem;line-height:1.5;letter-spacing:0.00938em;}",
        }
      }
      data-emotion="css ahj2mt-MuiTypography-root"
    />
    <p
      className="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
    >
      Open
    </p>
    <style
      dangerouslySetInnerHTML={
        Object {
          "__html": ".css-10tjo1u-MuiButton-root{font-family:\\"Roboto\\",\\"Helvetica\\",\\"Arial\\",sans-serif;font-weight:500;font-size:0.875rem;line-height:1.75;letter-spacing:0.02857em;text-transform:uppercase;min-width:64px;padding:5px 15px;border-radius:4px;-webkit-transition:background-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,box-shadow 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,border-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;transition:background-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,box-shadow 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,border-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;border:1px solid rgba(25, 118, 210, 0.5);color:#1976d2;}.css-10tjo1u-MuiButton-root:hover{-webkit-text-decoration:none;text-decoration:none;background-color:rgba(25, 118, 210, 0.04);border:1px solid #1976d2;}@media (hover: none){.css-10tjo1u-MuiButton-root:hover{background-color:transparent;}}.css-10tjo1u-MuiButton-root.Mui-disabled{color:rgba(0, 0, 0, 0.26);border:1px solid rgba(0, 0, 0, 0.12);}",
        }
      }
      data-emotion="css 10tjo1u-MuiButton-root"
    />
    <style
      dangerouslySetInnerHTML={
        Object {
          "__html": ".css-1rwt2y5-MuiButtonBase-root-MuiButton-root{display:-webkit-inline-box;display:-webkit-inline-flex;display:-ms-inline-flexbox;display:inline-flex;-webkit-align-items:center;-webkit-box-align:center;-ms-flex-align:center;align-items:center;-webkit-box-pack:center;-ms-flex-pack:center;-webkit-justify-content:center;justify-content:center;position:relative;box-sizing:border-box;-webkit-tap-highlight-color:transparent;background-color:transparent;outline:0;border:0;margin:0;border-radius:0;padding:0;cursor:pointer;-webkit-user-select:none;-moz-user-select:none;-ms-user-select:none;user-select:none;vertical-align:middle;-moz-appearance:none;-webkit-appearance:none;-webkit-text-decoration:none;text-decoration:none;color:inherit;font-family:\\"Roboto\\",\\"Helvetica\\",\\"Arial\\",sans-serif;font-weight:500;font-size:0.875rem;line-height:1.75;letter-spacing:0.02857em;text-transform:uppercase;min-width:64px;padding:5px 15px;border-radius:4px;-webkit-transition:background-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,box-shadow 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,border-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;transition:background-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,box-shadow 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,border-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;border:1px solid rgba(25, 118, 210, 0.5);color:#1976d2;}.css-1rwt2y5-MuiButtonBase-root-MuiButton-root::-moz-focus-inner{border-style:none;}.css-1rwt2y5-MuiButtonBase-root-MuiButton-root.Mui-disabled{pointer-events:none;cursor:default;}@media print{.css-1rwt2y5-MuiButtonBase-root-MuiButton-root{-webkit-print-color-adjust:exact;color-adjust:exact;}}.css-1rwt2y5-MuiButtonBase-root-MuiButton-root:hover{-webkit-text-decoration:none;text-decoration:none;background-color:rgba(25, 118, 210, 0.04);border:1px solid #1976d2;}@media (hover: none){.css-1rwt2y5-MuiButtonBase-root-MuiButton-root:hover{background-color:transparent;}}.css-1rwt2y5-MuiButtonBase-root-MuiButton-root.Mui-disabled{color:rgba(0, 0, 0, 0.26);border:1px solid rgba(0, 0, 0, 0.12);}",
        }
      }
      data-emotion="css 1rwt2y5-MuiButtonBase-root-MuiButton-root"
    />
    <button
      className="MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButtonBase-root  css-1rwt2y5-MuiButtonBase-root-MuiButton-root"
      disabled={false}
      onBlur={[Function]}
      onContextMenu={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
      type="submit"
    >
       
      <style
        dangerouslySetInnerHTML={
          Object {
            "__html": ".css-i4bv87-MuiSvgIcon-root{-webkit-user-select:none;-moz-user-select:none;-ms-user-select:none;user-select:none;width:1em;height:1em;display:inline-block;fill:currentColor;-webkit-flex-shrink:0;-ms-flex-negative:0;flex-shrink:0;-webkit-transition:fill 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;transition:fill 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;font-size:1.5rem;}",
          }
        }
        data-emotion="css i4bv87-MuiSvgIcon-root"
      />
      <svg
        aria-hidden={true}
        className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
        data-testid="SearchIcon"
        focusable="false"
        viewBox="0 0 24 24"
      >
        <path
          d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z"
        />
      </svg>
       
    </button>
  </form>
</div>
`;
